scout/parse/omim.py

```diff
@@ -379,15 +379,19 @@ def get_mim_phenotypes(genemap_lines: Iterable[str]) -> Dict[int, Any]:
     for entry in parse_genemap2(genemap_lines):
         hgnc_symbol = entry["hgnc_symbol"]
         for phenotype in entry["phenotypes"]:
+            #: For each phenotype extract mim
             mim_nr = phenotype["mim_number"]
-            if mim_nr in phenotypes_found:
-                phenotype_entry = phenotypes_found[mim_nr]
+            phenotype_id = f"OMIM:{mim_nr}"
+            if phenotype_id in phenotypes_found:
+                #: if mim is in found phenotypes, set this as the entry and union inheritance
+                phenotype_entry = phenotypes_found[f"OMIM:{mim_nr}"]
                 phenotype_entry["inheritance"] = phenotype_entry["inheritance"].union(
                     phenotype["inheritance"]
                 )
+                #: Only one on each row since gene based?
```

Comment:
Yes, Y think so!

Reply:
This was supposed to be removed in the cleanup 🥇